src/main/java/com/powsybl/caseserver/CaseException.java

```diff
@@ -35,7 +41,21 @@ private CaseException(Type type, String msg) {
         this.type = Objects.requireNonNull(type);
     }
 
-    public static CaseException createDirectoryAreadyExists(Path directory) {
+    public CaseException(Type type, String message, Exception e) {
```

Comment:
can be removed if you have the own with throwable no ?

Reply:
done

Reply:
done